spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/event/AssetDeleted.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.spi.event;
+
+import java.util.Objects;
+
+public class AssetDeleted extends Event {
+
+    private String id;
```

Comment:
is this supposed to be the `assetId`? if so, i'd rename it to avoid ambivalence: one could be lead to believe this were the event's ID.